src/main/java/com/airbnb/billow/AWSDatabase.java

```diff
@@ -103,10 +104,19 @@
                     ListTagsForResourceRequest tagsRequest = new ListTagsForResourceRequest()
                             .withResourceName(arnGenerator.rdsARN(regionName, instance));
 
-                    ListTagsForResourceResult tagsResult = client.listTagsForResource(tagsRequest);
-
-                    rdsBuilder.putAll(regionName, new RDSInstance(instance, tagsResult.getTagList()));
-
+                    try {
+                        ListTagsForResourceResult tagsResult = client.listTagsForResource(tagsRequest);
+
+                        rdsBuilder.putAll(regionName, new RDSInstance(instance, tagsResult.getTagList()));
+                    } catch(DBInstanceNotFoundException e) {
+                        // It is possible for an instance to disappear between when we got the list of instances and
+                        // when we go to find the instance's tags.
+                        log.warn("Unable to find RDS instance '" +
```

Comment:
So the traditional way to log with exceptions is:

    log.warn("message", exception);

That way the logger will provide a stacktrace if configured to do so.

Instead of building your string with + (which javac will internally convert to invocations of StringBuilder), it's best to use the logger's string formatting support. The idea there is that if a log won't be printed anyway (eg because the log level is disabled, you avoid work):

    log.warn("foo {} baz", "bar", exception);

Essentially, "{}" in the message format will consume parameters, and the last parameter that's an exception shouldn't be consumed by the message format as the logger will deal with it as it sees fit.